### ISSUE TITLE
Luk a kuša

### ISSUE THREAD

**Issue by pitkes22** [pexelnet/starving#51]
<h1>Naboje</h1>
- **Šip -** _Používajú sa do luku a kuše_
  - Mapping:`ARROW:0`
  - Max. itemou v stacku:`12`
  - Zriedkavosť :`UNCOMMON`
  - Kategória:`RANGED`

<h1>Zbrabne</h1>
- **Zvuk vystrelu**: `Nazov zbrane`_fire  
- **Zvuk prebitia**: `Nazov zbrane`_reload  

---

<h2>Luk</h2>

**Normalny luk v miencrafte**

---

<h2>Kuša</h2>
- Mapping:`DIAMOND:0` **Nenabita**
- Mapping:`COAL:0` **Nabita**
- Mapping:`COAL:1` **Nabita + mierenie**
  - Neda sa najsť.
- Max. itemou v stacku:`1`
- Zriedkavosť :`UNCOMMON`
- Kategória:`RANGED`
- Veľkosť zasobnika:`1`
  - Typ municie:`Šip`
- Spetny raz:`0`
- Rozptyl:`0.2/0.02`
- Čas prebijania:`3`
- Dostrel / rychlost projektilu:`80/5`
- Kadencia:`0`
- Hlucnost:`0`
- Ak hrač klikne(pravim alebo lavim) s Nenabitou kušou a ma v inventari nieaky naboj vymeni sa mu item na item nabitej kuše
- Pri kliknuti lavim s nabitou kušou sa item v ruke vymeni za item nabitej kuše (mierenie) s rovnakymi vlasnostami(Aj naopak z mierenia na normalny nabity)
- Keď hrač klikne pravim s nabitou košou vystreli
- Po vystrelei sa item opet vymeni za nenabitu kušu

**Comment by dobrakmato** [comment]
Mapping pridany do zoznamu.

**Comment by janscz** [comment]
Kuše není melee ale ranged weapon

**Comment by dobrakmato** [comment]
@janscz Ano, mame to zle, lebo nemame zatial kategoriu. Ak si najdes cas, tak mozes prist vyskusat strelne zbrane na server.

**Comment by janscz** [comment]
Ještě bych do ranged přidal Prak jako zbraň :D

**Comment by dobrakmato** [comment]
Ciastocne Implementovane.
